dali/kernels/reduce/reduce_gpu_impl.cuh

```diff
@@ -51,7 +52,7 @@ enum class ReductionKind {
   Block,   ///< Reduce contiguous samples separately
   Inner,   ///< Reduce inner dimension
   Middle,  ///< Reduce middle or outer dimension
-  None,    ///< No reduction - can happen when reduced dimension is degenerate
+  None,    ///< No reduction - can happen when reduced dimension has unit extent
```

Comment:
"degenerate" in this case was a misnomer - and possibly a trigger for forgetting about really degenerate cases.